src/context/IIIFResourceContext.tsx

```diff
@@ -25,7 +25,12 @@ export function IIIFContentProvider(props: {
     if (!props.overrides || !resource) {
       return resource;
     }
-    return Object.assign({}, resource, props.overrides || {});
+
+    const values = Object.fromEntries(
+      Object.entries(props.overrides).filter(([, value]) => typeof value !== 'undefined')
+    );
```

Comment:
This was a quick fix to allow undefined properties to be skipped in the overrides (from leva)